File: more_exercises/01_first_steps_in_coding/05_training_lab.py
width_meter = float(input())
height_meter = float(input())
width_cm = width_meter * 100
height_cm = height_meter * 100
row_seats = (height_cm - 100) // 70
column_seats = width_cm // 120
total_seats = row_seats * column_seats - 3

print(total_seats)

File: 05_JavaScriptBasics/03_MoreExercises/01_FirstStepsInCoding/05_trainingLab.js
function trainingLab(input){
    let widthMeters = Number(input[0])
    let lengthMeters = Number(input[1])

    let widthCentimeters = widthMeters * 100
    let lengthCentimeters = lengthMeters * 100
    let rowSeats = Math.floor((lengthCentimeters - 100)/70)
    let columnSeats = Math.floor(widthCentimeters/120)
    let totalSeats = rowSeats * columnSeats - 3

    console.log(totalSeats)
}

trainingLab(["15", "8.9"])
trainingLab(["8.4", "5.2"])
